BYT/Views/FoaasPreviewView.swift

```diff
@@ -241,4 +260,57 @@ class FoaasPreviewView: UIView {
     let view = UIView()
     return view
   }()
+    
+    internal lazy var doneButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.addTarget(self, action: #selector(doneButtonClicked(sender:)), for: UIControlEvents.touchUpInside)
+        button.setImage(UIImage(named: "done_button")!, for: .normal)
+        button.imageView?.layer.shadowColor = UIColor.black.cgColor
+        button.imageView?.layer.shadowOpacity = 0.8
+        button.imageView?.layer.shadowOffset = CGSize.zero
+        button.imageView?.layer.shadowRadius = 8
+        button.imageView?.clipsToBounds = false
+        
+        return button
+    }()
+    
+    internal lazy var backButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.addTarget(self, action: #selector(backButtonClicked(sender:)), for: UIControlEvents.touchUpInside)
+        button.setImage(UIImage(named: "back_button")!, for: .normal)
+        button.imageView?.layer.shadowColor = UIColor.black.cgColor
+        button.imageView?.layer.shadowOpacity = 0.8
+        button.imageView?.layer.shadowOffset = CGSize.zero
```

Comment:
adjust the y offsets to make the shadow is slightly lower than the button, like the light source was coming from above the status bar. also reduce the spread on the shadow to make it look closer to the view its over